### DISCUSSION TITLE
🖼 Fanart Competition #1

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#2553] [Art] [answered]
We are excited to announce a new brand new competition modality to showcase your creativity and love for Sunflower Land! 😍

For this competition, we encourage a wide range of art styles and techniques, not limited to pixel art. Feel free to explore your creativity and express yourself in any art style you choose!

Before submitting your artwork, we highly recommend that you read our fanart guidelines, which can be found here: 

https://docs.sunflower-land.com/contributing/competitions/fan-art-competition

## Theme 🎨

The theme for this month is: **Dawn Breaker Season** 🌄

Show us your creativity by creating something related to the chosen theme! Whether it's a character that you love, a scene that captivates you, or a moment that leaves a lasting impression, we want to see what you can come up with. Let your imagination soar and share your unique perspective with us! ❤

## Requirements ✏️

- AI-generated art will not be accepted.
- The art should be original and not plagiarised.
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/fan-art-competition#terms-and-conditions) of the game.

When submitting art, please include the following:

- Your Farm ID.
- Your Discord ID.
- A image of your art with good quality.
- A NPC that represents you. 

Please note that the inclusion of NPCs will not have any impact on the judging process of the competition. However, if you choose to provide an NPC, it will be displayed in-game alongside your art piece. If you do not include an NPC, a random one will be selected. Also, a Bumpkin version of your NPC would be appreciated, but not required. 

You can use this image to showcase your NPC:

![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/81132ce6-4863-4be7-a3ca-b5f78434ee60)

## Rewards 🏆

The winner will receive 1000 SFL and a runner-up will receive 500 SFL, however not only will the winning artwork be showcased on the official Sunflower Land Game social media channels and in-game, but it will also be seen by our vibrant community of players from around the world.

We also want to emphasize that even if you don't win, your design may still have the chance to be selected in the future. Our team will carefully review all submissions and may reach out to designers whose work stands out for future collaboration opportunities. Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅

Artists have until May 29th to submit their entries. After that, our art team will carefully review and analyze each submission to select the winners.

The announcement of the winners will take place during the first week of June.

So, what are you waiting for? Grab your brushes, pens, or digital tools and get creating! Show us what you're made of and join our community of talented artists in celebrating the vibrant world of Sunflower Land Game. We can't wait to see what you come up with!

Good luck to all participants! 🎉

## Example

Here's an amazing fanart example made by Vergelsxtn#8727 ! 

![image](https://user-images.githubusercontent.com/123522693/236883192-52605743-0478-427a-a9ab-e236e26d3750.png)

**Comment by Lvegas01** [comment]
Farm ID: 930
Discord: Luisve#5024

The darkness is coming

The darkness is coming and it is spreading all over the Sunflower Land, are you ready to defend yourself?
![sunfloewr](https://github.com/sunflower-land/sunflower-land/assets/34526371/f684a54e-9fe1-40fd-b577-7613e8f23c11)

**Comment by AxieFan1** [comment]
Farm ID: 6530
Discord: AxieFan1#5709

![Sfl farm 2](https://github.com/sunflower-land/sunflower-land/assets/124794436/7e475d91-17f1-445d-bcd7-e7c6b010d7d8)


Fan art 1   (Dawn breaks finally on my farm).

**Reply by felipereicampos** [reply]
I.A picture.... --'

**Comment by leafarogramac** [comment]
![_a256e126-c3ca-46f6-b9d0-09209b8abbf1](https://github.com/sunflower-land/sunflower-land/assets/83338231/5ba94fc1-3e53-4bd5-b807-f344b53c9c40)

Farm ID: 129291
Discord: leafar13#6745

**Reply by felipereicampos** [reply]
I.A picture.....

**Reply by leafarogramac** [reply]
where is it described that you can't use AI? I didn't read about it.

**Reply by Alaindrea** [reply]
Requirements ✏️
AI-generated art will not be accepted.

**Reply by CaioCastelo** [reply]
ia...

**Comment by yo121-eth** [comment]
Farm ID: 576
Discord: TreasureGoblin.wallet#1582
chaos after war
![sfl pint](https://github.com/sunflower-land/sunflower-land/assets/102257243/6e720a4d-1000-413d-86d9-cedcf7ffbd33)

**Reply by felipereicampos** [reply]
I.A picture ............ boring

**Reply by yo121-eth** [reply]
> 
Yes, it's an AI GTP image... if you have any excellent ideas, plz share them with us.
Now you look like a right Lemon...

**Reply by leafarogramac** [reply]
that's right.. the guy commented the same crap on my post....
boring thing, do it better and post it there, right?!

and congrats yo121-eth chose a nice image...good luck.

**Reply by Alaindrea** [reply]
Requirements ✏️
AI-generated art will not be accepted.

**Reply by CaioCastelo** [reply]
ia...

**Comment by CiberusNovo** [comment]
Farm ID: 143188
Discord: Ciberus#1050
                The darkness is here! 
             Time to defend your land. 
    I'll be ready for whatever comes next.
First I must go north and visit Marcus and his family,  I just hope it's not too late.
![DefendYourSunflowers](https://github.com/sunflower-land/sunflower-land/assets/133623205/c0c9be82-c14d-426b-88d7-631705632bf6)

**Comment by akosigau** [comment]
Land #5521
Gau #7355
![Land #5521](https://github.com/sunflower-land/sunflower-land/assets/71237827/03bccc1c-7184-4816-a34c-198772673169)

**Comment by Jakesmoked-141829** [comment]
hi,
i was sketchin for fun then i share it just in case 
Farm ID - 141829
Discord ID - jakesmoked#0563


![v2](https://github.com/sunflower-land/sunflower-land/assets/124737947/f168b018-dd4c-4a51-b9dc-8fda0e1324ec)


NPC : ![N0IXVstHEX](https://github.com/sunflower-land/sunflower-land/assets/124737947/d526aaea-a8b4-4257-80a4-cfc814543fbb)

**Comment by bayselb** [comment]
Farm ID: 154354
Discord: Bayselb#0582

Super Eggplant Bros
![image](https://github.com/sunflower-land/sunflower-land/assets/133677194/c1c3a95c-a84d-4657-98dc-be12bb4f91af)

**Comment by lukkarj** [comment]
Farm ID # 147273
Discord ID # Lukkarj#2159


![Bumpkin-art-2](https://github.com/sunflower-land/sunflower-land/assets/98267583/73fb1815-297c-4a81-b839-a8ed86dec576)

**Comment by rendito69** [comment]
Farm ID # 40716
Discord ID # rendito#5133
![sunflower land- sun flower warrior](https://github.com/sunflower-land/sunflower-land/assets/32850047/f95416e7-a308-444e-862c-e67c468bc215)
Bumpkin #22171

**Comment by jujubasfrescas** [comment]
![_fc2513cf-7214-4540-bda7-4f61684fe4bf](https://github.com/sunflower-land/sunflower-land/assets/132146194/da195318-371a-4d94-9815-8974b8ec3e26)
Farm ID: 69553
Discord ID: joaotib#0697

**Comment by Netherzapdos** [comment]
To all future artists planning on using AI..

at least make it look related to the game, you guys just slap a sunflower on it and call it a day 🤣 

Gotta put in SOME effort at least like come on... It's not that hard to describe what SFL looks like 😉

**Reply by ddx1458** [reply]
how to post for drawing?)

**Comment by BirbSunflower** [comment]
For those who did not read the announcement:

We would like to inform that AI-generated art will not be accepted for our Fan Art Competition. 

While we recognize the potential of AI technology to produce impressive visual pieces, we strongly believe that the Fan Art Competition should showcase the creative talent and personal expression of human artists. We want to celebrate the hard work, dedication, and unique artistic styles of our community!

We appreciate your understanding and look forward to receiving your original submissions that reflect your creativity and artistic vision! :heart:

https://discord.com/channels/880987707214544966/1072288889273847889/1107827408515633263

**Comment by yapiciemre** [comment]
`Farm ID : 7446 `
`Discord : Agnes#1353`

# **Information**
This image was drawn using Adobe Photoshop. The files are available.

# **A NIGHT AT THE TIME OF DAWNBREAKER**
![ForFanArtCompetition-1](https://github.com/sunflower-land/sunflower-land/assets/55670141/82be0417-cd6a-4ec0-9624-89217f72d431)

**Comment by mmaxxpeinn** [comment]
![art_compe_db](https://github.com/sunflower-land/sunflower-land/assets/120295844/d579362e-a9bb-4903-9ce6-67909c4b488e)

in-game NPC:
![maxx_npc_land_preview](https://github.com/sunflower-land/sunflower-land/assets/120295844/6d060562-f5d2-46cb-8bf4-70baeac09d7a)
minified ver.: 
![maxx_npc_preview](https://github.com/sunflower-land/sunflower-land/assets/120295844/93d1cde9-d168-4ce0-9f66-3aeb85fdf15b)
NPC sprite sheet:
![maxx_npc_sheet (1)](https://github.com/sunflower-land/sunflower-land/assets/120295844/1f848022-6b51-4333-b681-7035650f164c)
Hair/glasses sprite sheet:
![maxx_hair_glass_npc_sheet](https://github.com/sunflower-land/sunflower-land/assets/120295844/38c8cd69-ee63-4a73-8d42-5aabb99edd75)
Shirt sprite sheet:
![shirt_npc_sheet](https://github.com/sunflower-land/sunflower-land/assets/120295844/710b6718-701d-4eed-bd72-89c246986fc4)

`Farm ID: 82122`
`DC: mmaxxpeinn#9351`

**Comment by EmpadinhaDoMal26** [comment]
Farm id: 60926
discord id: EmpadinhaDoMal#1393

### the light came back!


![empadinhadomal](https://github.com/sunflower-land/sunflower-land/assets/134020787/aba19099-7a9f-43cf-bcf3-b906a6edd323)

**Comment by hpkamikaze** [comment]
Farm ID: 68095
Discord: hpanderson#7902 
![finalarte](https://github.com/sunflower-land/sunflower-land/assets/134032805/8f638141-121b-4841-b3a6-eccac07b0207)

**Comment by Armir-NFT** [comment]
Farm ID: 139443
Discord: Armir#4056
"Conquer the darkness"
![all](https://github.com/sunflower-land/sunflower-land/assets/114429594/034de984-8dbf-4cc0-8f35-4a0a8bca1a58)
![Screenshot 2023-05-22 001515](https://github.com/sunflower-land/sunflower-land/assets/114429594/50cb3d33-c7ae-45bd-a5b0-79dc7e4c22aa)

**Reply by On1el** [reply]
I like it very much

**Reply by YamiYamiKokiKopi** [reply]
win

**Reply by kylaAdventures** [reply]
Really nice.

**Comment by kylaAdventures** [comment]
Hello,
here is my entry for the event,![IMG_3636](https://github.com/sunflower-land/sunflower-land/assets/133995914/378a1ae0-8bdd-4c13-b6de-52e2fce59ec3)
Farm ID:15425
Discord ID:shinobigoutcoco#8582

**Comment by BrunoLuna97** [comment]
Hi all! Here is my contribution to the contest 😃 My inspiration is the Japanese festivals 😄 I create with Cinema 4D and Photoshop
Farm ID: 14386
Discord ID: Animabob#8837

![Sunflower_Contest](https://github.com/sunflower-land/sunflower-land/assets/78119767/85a0f120-df38-46a6-a8dc-cc786d4c38a5)

**Comment by NancyTang50** [comment]
### **My mushroom Dawn Breaker Friends**
Enjoying the weather with Sophia and Bella ❤️🌅

**Farm ID: _18894_**
**Discord ID: _DevNancyStudent#6982_**

![My mushroom dawn breaker friends](https://github.com/sunflower-land/sunflower-land/assets/38754670/c61bf9c4-36db-4298-9763-ff3b35eb59a0)

**Comment by Cheepeepa** [comment]
Farm Id : 11605
Discord ID: Cheep#6799

inspired by : "Hmmph! Go Away..."
![github](https://github.com/sunflower-land/sunflower-land/assets/134818096/e760ba13-0c7d-453b-9910-68220ec92912)

**Comment by Je3an** [comment]
ID discord: Je3an ☕🍪#5448
Land #162279

**illustration inspired in my bumpkin**

![Ilustración5](https://github.com/sunflower-land/sunflower-land/assets/134823091/1305a681-0a8c-4ded-8d60-06fa9526f23e)

**Reply by JeffMontil** [reply]
Niceeee

**Reply by xMan17x** [reply]
its the best!

**Reply by 2014online2014** [reply]
Sweet!

**Comment by jjaleco** [comment, marked as answer]
![sunflower](https://github.com/sunflower-land/sunflower-land/assets/133828062/fa51d74c-b47b-4188-9746-22813d61f332)
farm id: 144799
discord id: palisman#7432

tradigi art (traditional + digital art)
done using color pencil and water color. The lightings and some adjustments are done digitally. 

npc 
![npc](https://github.com/sunflower-land/sunflower-land/assets/133828062/71318979-d60c-46f1-8540-9f8b4d3e9f5d)

**Comment by Dreamoon666** [comment]
Farm ID : 45114
Discord ID : DevilEvil#8440

FanArt Title : Tales of the Dawn Breaker ...

![SunFlower Land FanArt Dawn Breaker Event](https://github.com/sunflower-land/sunflower-land/assets/134853083/5e0dddee-02ad-490a-9c41-4bb9ccb7ea84)

![NPC](https://github.com/sunflower-land/sunflower-land/assets/134853083/27ae23b1-5bed-4627-aff6-8a1fe1ef565c)

![Bumpkin](https://github.com/sunflower-land/sunflower-land/assets/134853083/1d294a66-f3e9-46db-bc87-d8e83dc16276)

**Reply by CaioCastelo** [reply]
ia

**Reply by Dreamoon666** [reply]
no its not IA its a Digital Painting and i can prove it

**Comment by catnoir2604** [comment]
Farm ID: 8149
Discord: Milagros#7008
![photo_5161310389592501137_y](https://github.com/sunflower-land/sunflower-land/assets/115124640/a963a132-caed-41ad-8318-099cef6eb45f)

**Comment by archila79** [comment]
farm: #143991
archila#9474
![Proyecto nuevo (2)](https://github.com/sunflower-land/sunflower-land/assets/115169880/11b2c58e-fd2c-49e9-a2fc-c85f26e063fc)

**Reply by NycksonV** [reply]
IA, art?!

**Reply by archila79** [reply]
no, that's called photoshop and you have to study to make these styles, AI only serves to belittle the work of a designer and people who don't know about these topics only know how to respond like you...

**Comment by esp123** [comment]
Hello!
![findingthedawn](https://github.com/sunflower-land/sunflower-land/assets/74337946/9b46ffd7-f769-4824-847c-54d9db765c55)
![dawnchill](https://github.com/sunflower-land/sunflower-land/assets/74337946/161d3bc2-209d-4c81-b099-ce61c01f4ca7)

NPC:
![mariooooo](https://github.com/sunflower-land/sunflower-land/assets/74337946/8cd9ae23-79dd-43d0-8e41-20aaac4376d6)


Discord: deeefault#0033
Farm ID: 80399

**Comment by arconycompany** [comment]
Discord : Arcony#1641
Farm ID: 1939

Dawn Ticket Secret :

![image](https://github.com/sunflower-land/sunflower-land/assets/70727157/addce810-99f0-44de-9ef6-f6f7d4e62d4c)

**Comment by PurpleDrvnk** [comment]
Discord: PurpleDrvnk#8703
FarmID: 155678

My Submission: The Eggplant Bros (Dont trust these guys!)

![SFLARTCOMP](https://github.com/sunflower-land/sunflower-land/assets/127863871/5a71635e-f480-469e-adc8-bcd7b945a7b5)

My NPC:


![SFLARTCOMPNPC-export](https://github.com/sunflower-land/sunflower-land/assets/127863871/b579b04d-56f1-4b96-8df4-c7a7ab3a785a)


NPC Sheet:

![SFLARTCOMPNPC2](https://github.com/sunflower-land/sunflower-land/assets/127863871/209df20e-50f2-4351-a228-d47c5ad2a5c3)

**Comment by NycksonV** [comment]
`Farm ID: 136998`
`Discord: Nyck#4168`

## Fan art
![PosterFinal](https://github.com/sunflower-land/sunflower-land/assets/118012559/9499e718-2644-4fdb-b3a1-035c8c2d903f)
The Lith tries to destroy the dawn in the land of Sunflower!

##

## Animation
(Watch until End) A small animation of the poster of approximately 10 seconds
![TEste1](https://github.com/sunflower-land/sunflower-land/assets/118012559/c2ac1f3f-073a-48d3-8b38-025295662fb1)

##

## Poro little Bumpkin
This is me  u,u
![Porinho](https://github.com/sunflower-land/sunflower-land/assets/118012559/371e3c76-4145-4472-9187-ac866953ee4c)


| Sprite |
| --- |
![Porinho](https://github.com/sunflower-land/sunflower-land/assets/118012559/f330cb07-c963-4e8a-b7d1-15dd2ff8b59d)


##

## Sprite  ![Poro0](https://github.com/sunflower-land/sunflower-land/assets/118012559/18a4890f-0173-4616-ac74-d1983561d7cd)

##

**Reply by esp123** [reply]
This is so nicee poro :100:

**Reply by NycksonV** [reply]
thanks my dear

**Reply by NycksonV** [reply]
I'm glad you like S2

**Comment by LTNGx39** [comment]
### **"Mushroom Island, before the Dawn Breaker"**

**Farm ID: 161769
Discord: Neonlight#6514**

I wanted to add more things but I am satisfied with the final result, I hope you like it

![Final Render 3K](https://github.com/sunflower-land/sunflower-land/assets/131133076/762deabf-0bc9-4f7d-b1a2-1adcf6060970)

**Comment by esp123** [comment]
![Untitled 05-30-2023 07-45-45 (1)](https://github.com/sunflower-land/sunflower-land/assets/74337946/7594ef4f-7725-41bc-8ecd-e2e4bd84659f)
*A wild mushroom appears*

**Comment by bottlemani11** [comment]
![WhatsApp Image 2023-06-07 at 4 39 41 PM](https://github.com/sunflower-land/sunflower-land/assets/126830260/7ed7f96d-5b00-42b1-ba98-7033825f382f)
My 5yr old niece have drawn this for me <3
Land id: 130596
